src/ethereum/trace.py

```diff
@@ -0,0 +1,86 @@
+"""
+.. _trace:
+
+EVM Trace
+^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Defines the functions required for creating evm traces during execution.
+"""
+
+from typing import Any
+
+
+def capture_tx_start(env: Any) -> None:
+    """
+    Capture the state at the beginning of a transaction.
+    """
+    pass
+
+
+def capture_tx_end(
```

Comment:
Pretty insignificant, but what do you think of `trace_tx_end`, `trace_precompile_start`, etc instead of `capture_*`?

Reply:
Since we are moving everything under a single function `evm_trace`, this change is no longer needed imo.